next-tavla/pages/organizations/index.tsx

```diff
@@ -24,17 +27,31 @@ export async function getServerSideProps({
     return {
         props: {
             loggedIn: user !== null,
+            organizations: await getOrganizationsWithUser(user.uid),
+            userId: user.uid,
         },
     }
 }
 
-function OrganizationsPage({ loggedIn }: { loggedIn: boolean }) {
+function OrganizationsPage({
+    loggedIn,
+    organizations,
+    userId,
+}: {
+    loggedIn: boolean
+    organizations: TOrganization[]
+    userId: TUserID
+}) {
     return (
         <div className={classes.root}>
             <AdminHeader loggedIn={loggedIn} />
             <Contrast>
                 <ToastProvider>
-                    <Organizations />
+                    <UserOrganizationsContext.Provider
+                        value={{ userId, organizations }}
```

Comment:
Litt usikker på hele strukturen på siden og contexten, men må jo på en eller annen måte ha med seg `userId` for å finne relasjonen mellom brukeren og organisasjonen.

Kom gjerne med andre forslag 🤠